webapp/portlet/src/main/webapp/vue-apps/user-setting-notifications/components/UserSettingNotificationPlugin.vue

```diff
@@ -20,7 +20,9 @@
         <v-list-item-title class="text-wrap">
           <template v-if="enabledDigestLabel">
             <v-chip class="ma-2" color="primary">
-              {{ enabledDigestLabel }}
+              <span :class="{ 'mobile-chip-ellipsis' : isMobile }">
```

Comment:
Can you explain this modification ? Why we need to add the isMobile ?

Reply:
Problem fixed

Reply:
it was a problem of updating in the branch